projects/Mallard/src/authentication/AccessContext.tsx

```diff
@@ -1,157 +1,151 @@
+const signOutIdentity = identity.signOut.bind(identity);
 
 const AccessProvider = ({
-    children,
-    onIdentityStatusChange = () => {},
+	children,
+	onIdentityStatusChange = () => {},
 }: {
-    children: React.ReactNode
-    onIdentityStatusChange?: (idAttempt: AnyAttempt<IdentityAuthData>) => void
+	children: React.ReactNode;
+	onIdentityStatusChange?: (idAttempt: AnyAttempt<IdentityAuthData>) => void;
 }) => {
-    const [attempt, setAttempt] = useState<AnyAttempt<AttemptType>>(
-        controller.getAttempt(),
-    )
-    const [idAuth, setIdAuth] = useState<AnyAttempt<IdentityAuthData>>(
-        controller.authorizerMap.identity.getAttempt(),
-    )
-    const [casAuth, setCASAuth] = useState<AnyAttempt<CASExpiry>>(
-        controller.authorizerMap.cas.getAttempt(),
-    )
-    const [iapAuth, setIAPAuth] = useState<AnyAttempt<ReceiptIOS>>(
-        controller.authorizerMap.iap.getAttempt(),
-    )
-    const client = useApolloClient()
+	const [attempt, setAttempt] = useState<AnyAttempt<AttemptType>>(
+		controller.getAttempt(),
+	);
+	const [idAuth, setIdAuth] = useState<AnyAttempt<IdentityAuthData>>(
+		controller.authorizerMap.identity.getAttempt(),
+	);
+	const [casAuth, setCASAuth] = useState<AnyAttempt<CASExpiry>>(
+		controller.authorizerMap.cas.getAttempt(),
+	);
+	const [iapAuth, setIAPAuth] = useState<AnyAttempt<ReceiptIOS>>(
+		controller.authorizerMap.iap.getAttempt(),
+	);
+	const client = useApolloClient();
 
-    useEffect(() => {
-        const unsubController = controller.subscribe(setAttempt)
-        const unsubIdentity = controller.authorizerMap.identity.subscribe(
-            attempt => {
-                setIdAuth(attempt)
-                onIdentityStatusChange(attempt)
-            },
-        )
-        const unsubCAS = controller.authorizerMap.cas.subscribe(setCASAuth)
-        const unsubIAP = controller.authorizerMap.iap.subscribe(setIAPAuth)
-        client
-            .watchQuery({
-                query: gql(
-                    '{ netInfo @client { isConnected @client, isPoorConnection @client } }',
-                ),
-            })
-            .subscribe(res => {
-                controller.handleConnectionStatusChanged(
-                    res.data.netInfo.isConnected,
-                    res.data.netInfo.isPoorConnection,
-                )
-            })
-        return () => {
-            unsubController()
-            unsubIdentity()
-            unsubCAS()
-            unsubIAP()
-        }
-        // eslint-disable-next-line react-hooks/exhaustive-deps
-    }, [])
+	useEffect(() => {
+		const unsubController = controller.subscribe(setAttempt);
+		const unsubIdentity = controller.authorizerMap.identity.subscribe(
+			(attempt) => {
+				setIdAuth(attempt);
+				onIdentityStatusChange(attempt);
+			},
+		);
+		const unsubCAS = controller.authorizerMap.cas.subscribe(setCASAuth);
+		const unsubIAP = controller.authorizerMap.iap.subscribe(setIAPAuth);
+		client
+			.watchQuery({
```

Comment:
Not sure what rule it applies but I thought `fix` would combine line 105 and 106 into single line

Reply:
I think prettier keeps all the subsequent functions at the same left-alignment, so `watchQuery` and `subscribe` both have the same left hand margin.